cocos/base/ccUTF8.cpp

```diff
@@ -526,7 +526,8 @@ long cc_utf8_strlen (const char * p, int /*max*/)
 unsigned int cc_utf8_find_last_not_char(const std::vector<unsigned short>& str, unsigned short c)
 {
     std::vector<char16_t> char16Vector;
-    for (const auto& e : str)
+    char16Vector.reserve(str.size());
+for (const auto& e : str)
```

Comment:
error indent

Reply:
Fixed. How do  I even missed that?

Reply:
@minggo is it possible to create .clang-format file for cocos2d-x and use automated formatting everywhere?

Reply:
Yep, sure. Would you please add one?